.github/workflows/snapm.yml

```
@@ -120,4 +122,55 @@ jobs:
+                          "pc-i440fx-*",
+                          "pc-q35-*"
+                      ]
+                  }
+              ],
+              "features": [],
+              "tags": []
+          }
+          EOF
+          sudo cp /tmp/20-seabios.json /usr/share/qemu/firmware/20-seabios.json
+      - name: Run virt-tests for target
+        run: >
```

actionlint (1.7.7)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)